source/includes/kubernetes/_k8_daemonsets.md

```diff
@@ -76,3 +76,73 @@ Retrieve a daemon set and all its info in a given [environment](#administration-
 
 Note that the list is not complete, since it is refering to the [kubernetes api details](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md).
+
+<!-------------------- CREATE DAEMON SET -------------------->
+
+### Create a daemon set
+```shell
+curl -X POST \
+  -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/pods"
+  Content-Type: application/json
+  {
+  "apiVersion": "apps/v1",
+  "metadata": {
+    "name": "daemonset-name",
+    "namespace": "default"
+  },
+  "spec": {
+    "selector": {
+      "matchLabels": {
+        "name": "fluentd-elasticsearch"
+      }
+    },
+    "template": {
+      "metadata": {
+        "labels": {
+          "name": "fluentd-elasticsearch"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "name": "fluentd-elasticsearch",
+            "image": "quay.io/fluentd_elasticsearch/fluentd:v2.5.2"
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "1542bd45-4732-419b-87b6-4ea6ec695c2b",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/daemonsets</code>
+
+Create a daemon set in a given [environment](#administration-environments).
+
+| Attributes                                 | &nbsp;                                                                    |
+| ------------------------------------------ | --------------------------------------------------------------------------|
+| `apiVersion` <br/> _string_                | The api version (versioned schema) of the daemon set                      |
+| `metadata` <br/>_object_                   | The metadata of the daemon set                                            |
+| `metadata.name` <br/>_string_              | The name of the daemon set                                                |
+| `metadata.namespace` <br/>_string_         | The namespace in which the daemon set is created                          |
+| `spec`<br/>_object_                        | The specification used to create and run the daemon set                   |
+| `spec.selector.matchLabels`<br/>_object_   | The key value pairs retrieved by a label query from a daemon set          |
+| `spec.template`<br/>_object_               | The data a daemon set's pod should have when created                      |
+| `spec.spec`<br/>*object*                   | The specification used to create and run the pod(s) within the daemon set |
+
+Return value:
+
+| Attributes                 | &nbsp;                                              |
+---------------------------- | ----------------------------------------------------|
+| `taskId` <br/>*string*     | The id corresponding to the daemon set create task. |
```

Comment:
How about - `The id corresponding to the create daemon set task.`? Feels like it slightly reads better. But you can leave is as is if you feel otherwise.